bot/exts/filters/filtering.py

```diff
@@ -628,6 +634,22 @@ async def delete_offensive_msg(self, msg: Mapping[str, str]) -> None:
         await self.bot.api_client.delete(f'bot/offensive-messages/{msg["id"]}')
         log.info(f"Deleted the offensive message with id {msg['id']}.")
 
+    @staticmethod
+    def remove_invisible_chars(string: str) -> str:
+        """
+        Remove invisible characters from `string`.
+
+        Removed characters:
+
+        - mongolian vowel separator
+        - zero width space
+        - zero width non-joiner
+        - zero width joiner
+        - word joiner
+        - zero width non-breaking space
+        """
+        return re.sub("[\u180e\u200b\u200c\u200d\u2060\ufeff]", "", string)
```

Comment:
Could avoid the docstring commenting on the character names if named escapes would be used here